modules/client/src/tokenVoting/internal/client/methods.ts

```diff
@@ -738,4 +750,54 @@ export class TokenVotingClientMethods extends ClientCore
     }
     return null;
   }
+
+  /**
+   * Checks if the given token is compatible with the TokenVoting plugin
+   *
+   * @param {string} tokenAddress
+   * @return {*}  {Promise<boolean>}
+   * @memberof TokenVotingClientMethods
+   */
+  public async isTokenGovernanceCompatible(
+    tokenAddress: string,
+  ): Promise<boolean> {
```

Comment:
IMO, this function should return three things:
- token doesn't need wrapping
- needs wrapping to be compatible
- token cannot be used